src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0">
```

Comment:
```suggestion
    <div class="stars stars--0">
```

Each stars block should have a main class `stars` and class-modifier